mojo-trifacta-udf/src/main/java/common/ai/h2o/dai/mojo/MojoScoringInterface.java

```
@@ -0,0 +1,132 @@
+package common.ai.h2o.dai.mojo;
+
+
+import ai.h2o.mojos.runtime.frame.MojoFrameBuilder;
+import ai.h2o.mojos.runtime.frame.MojoFrameMeta;
+import ai.h2o.mojos.runtime.frame.MojoRowBuilder;
+import ai.h2o.mojos.runtime.frame.MojoFrame;
+import ai.h2o.mojos.runtime.MojoPipeline;
+import com.google.gson.JsonObject;
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.HashMap;
+
+
+public class MojoScoringInterface {
+
+    public String transformLocal(MojoPipeline mojo, MojoFrameMeta mojoFrameMeta, String inputRow, String delimiter) {
+        try {
+            String[] rowArray = inputRow.split(delimiter);
+            MojoFrame inputFrame = generateInputMojoFrame(mojo, mojoFrameMeta, rowArray);
+            MojoFrame outputFrame = mojo.transform(inputFrame);
+            return generateResponse(outputFrame);
+        } catch (Exception e) {
+            System.out.println("EXCEPTION: ");
```

Should we let it bubble up, wrap or such instead of silencing and returning a `null` that will bite us later on?